### DISCUSSION TITLE
AlDente 1.24

### DISCUSSION THREAD

**Discussion by davidwernhart** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1109] [General]
**Linguini**

- [All] Fixed a bug that caused MacBooks with Intel Mode enabled to not maintain the charge limit when AlDente was closed. (This time for real, we promise!)
- [All] Fixed a bug that caused the app to crash when the "Schedule" tab was selected in AlDente's settings.
- [All] Fixed a bug that caused the app to crash on Apple Silicon MacBooks running macOS Big Sur or Sonoma.
- [All] Improved VoiceOver usability.
- [Pro] Complete overhaul of the "Sailing Mode" feature. Read more about in on our blog: [Feature Explanation: Sailing Mode](https://apphousekitchen.com/feature-explanation-sailing-mode/).
- [Pro] Added a new visual indicator for the sailing range in the status bar.
- [Pro] The Power Flow feature is out of Beta!
- [Pro] Tweaked the status bar colors for better readability of the status icons.
- [Pro] Fixed a bug that caused "Stop Charging when Sleeping" to not work properly in some scenarios.
- [Pro] Fixed a bug that caused MacBooks in Clamshell mode to not go to sleep properly if sleep was disabled beforehand.

With peace & love from Vienna,  
AppHouseKitchen

<hr /><em>This discussion was created from the release <a href='https://github.com/AppHouseKitchen/AlDente-Charge-Limiter/releases/tag/1.24'>AlDente 1.24</a>.</em>

**Comment by bayonet75** [comment]
Hi
DrWeb found the problem: Tool.Mac.BtcMine.9964
![Screen Shot 2023-11-26 at 8 46 32 PM](https://github.com/AppHouseKitchen/AlDente-Charge-Limiter/assets/145794156/1f9d33a8-f642-4d77-982f-28a5659de646)

**Reply by MatthiasKerbl** [reply]
Hi @bayonet75,

Thanks for letting us know. Also, there is already a thread about this open #1108 with a statement from our side.

Best,
Matthias

**Reply by bayonet75** [reply]
Thanks!

**Reply by MatthiasKerbl** [reply]
Hi @bayonet75,

Good news, we have just released AlDente 1.24.1 which does not include the Blessed library anymore. Therefore, AlDente does not get reported by DrWeb or any other virus scanner on VirusTotal anymore.

Best,
Matthias
